lib/IRGen/IRGen.cpp

```diff
@@ -101,6 +101,15 @@ static void addThreadSanitizerPass(const PassManagerBuilder &Builder,
   PM.add(createThreadSanitizerPass());
 }
 
+template <SanitizerCoverageOptions::Type sanitizerCoverageType>
+static void addSanitizerCoveragePass(const PassManagerBuilder &Builder,
+                                     legacy::PassManagerBase &PM) {
+  SanitizerCoverageOptions opts;
+  opts.CoverageType = sanitizerCoverageType;
+  assert(opts.CoverageType != SanitizerCoverageOptions::SCK_None);
```

Comment:
You should be able to make this a `static_assert`, since it's a template parameter.